### PR TITLE
Add package references to project file

### PR DESCRIPTION
**Pull request by eNeRGy164**
Add the ability to get package references with name and (optional) version.
Helps with #110

### CODE REVIEW

File: src/Buildalyzer/Construction/ProjectFile.cs [hunk 1]
```diff
@@ -103,6 +103,11 @@ internal ProjectFile(string path, IProjectTransformer transformer)
         /// </summary>
         public bool ContainsPackageReferences => _projectElement.GetDescendants(ProjectFileNames.PackageReference).Any();
 
+        /// <summary>
+        /// The list of <c>PackageReference</c> items in the project file.
+        /// </summary>
+        public IReadOnlyList<PackageReference> PackageReference => _projectElement.GetDescendants(ProjectFileNames.PackageReference).Select(s => new PackageReference(s)).ToList();
```
Comment (glennawatson) [inline src/Buildalyzer/Construction/ProjectFile.cs]: Should be plural? So PackageReferences

**Comment by glennawatson** [comment]
+1 this one, have a project at the moment that could benefit from it.

File: src/Buildalyzer/Construction/ProjectFile.cs (hunk 1 above)
Comment (eNeRGy164) [reply src/Buildalyzer/Construction/ProjectFile.cs]: Oops, yes indeed

**Comment by daveaglick** [comment]
Thanks! Going to get a new release with this change out the door momentarily.

**Pull request merged**